src/web/services/string-helper.ts

```diff
@@ -3,6 +3,12 @@
  */
 export class StringHelper {
 
+  static readonly REGEX_FOR_ANOYNMOUS_PARTICIPANT: RegExp = new RegExp(
```

Comment:
typo, please fix the other parts of the code where this variable 
```suggestion
  static readonly REGEX_FOR_ANONYMOUS_PARTICIPANT: RegExp = new RegExp(
```is used.